server/controllers/common/handlerAllProducts.js

```diff
@@ -0,0 +1,12 @@
+const { getAllProducts } = require('../../database/quieres');
```

Comment:
there is a typo in quieres ,
change it to 'queries' and modify all related files and folder